doc/continuous_integration/NightlyBuilds.md

```diff
@@ -0,0 +1,20 @@
+# Nightly builds in Windows CI description
+This document describes how nightly builds are done in Contrail Windows CI.
+# Jenkins jobs: ci-contrail-windows-nightly-*
+* There are two jobs responsible for nightly building in CI:
```

Comment:
there is `nightly build` in almost every line at the beginning, don't you think it's kinda redundant?